fetch.bs

```diff
@@ -9062,6 +9062,107 @@ done only by navigations). The <a>fetch controller</a> is also used to
 <a for=request>redirect mode</a> set to "<code>manual</code>".
 
 
+<h3 id=fetch-elsewhere-security>Same-origin policy when using fetch</h3>
+
+<p>The Fetch standard enforces some of the mechanisms that constitute the Same Origin Policy (SOP),
+which is a critical aspect of web security. Specifically, Fetch imposes SOP restrictions on network
+access.
+
+<p>In a nutshell, when a user accesses a certain origin using their browser or other user agent,
+they don't expect that the user agent implicitly grants that <a for=/>origin</a> full access to
+their network, which could, for example, be an intranet, despite the user agent itself having that
+access.
```

Comment:
This paragraph may be too focused on users. What are specs using Fetch supposed to do to help with the SOP? It seems like their job is to ensure that resources are fetched with one of the [request modes](https://fetch.spec.whatwg.org/#concept-request-mode), which will ensure that one origin can't get illegitimate access to resources protected by credentials or network identity.

Reply:
It was actually important for me to be focused on users! But point taken.